### DISCUSSION TITLE
Helper in Markdown?

### DISCUSSION THREAD

**Discussion by simplerethink** [lumeland/lume#355] [Q&A] [answered]
I created a helper [similar to the docs](https://lume.land/docs/configuration/filters/#helpers)

```
// _config.ts

site.helper("myUppercase", (body) => body.toUpperCase(), {
    type: "tag",
    body: "true",
});
```
```
// index.njk

{% myUppercase %}
    Hello, this should be uppercase
{% endmyUppercase %}
```

This work fine.

However, I'd like to create a helper (shortcode) for markdown files.

```
// test.md

{% myUppercase %}
Hello, this should be uppercase
{% endmyUppercase %}
```
This produces output of `{% myUppercase %} Hello, this should be uppercase`

**Comment by oscarotero** [comment, marked as answer]
Yes, you're using nunjucks sintax in a markdown file, so it won't work.
But you can configure the render engine to use both. See this: https://lume.land/docs/core/multiple-template-engines/

**Reply by simplerethink** [reply]
Thank you. I didn't get far enough into the docs. It's working now. Thanks!
